ci/kokoro/docker/build-in-docker-bazel.sh

```diff
@@ -36,12 +36,12 @@ source "${PROJECT_ROOT}/ci/colors.sh"
 # ci/Dockerfile.* build scripts.
 
 echo
-echo "${COLOR_YELLOW}Starting docker build $(date) with ${NCPU} cores${COLOR_RESET}"
+echo "${COLOR_YELLOW}$(date -u): Starting docker build $(date) with ${NCPU} cores${COLOR_RESET}"
```

Comment:
Did you mean to remove that second `date`?

Reply:
Yes, thanks.  Fixed.